packages/rocketchat-models/server/models/LivechatDepartmentAgents.js

```diff
@@ -101,6 +101,28 @@ export class LivechatDepartmentAgents extends Base {
 		}
 	}
 
+	departmentIsOnline(departmentId) {
```

Comment:
Why did you duplicated the code instead of call `getOnlineForDepartment` and check the result?

Reply:
Because as you described before, on the `getOnlineForDepartment` method was possible to get 2 types of return(Array or Cursor) and on the old implementation, when returning an Array, an error was being raised when calling `onlineAgents.count()` function.
So, due to that, I decided to create a new method returning `boolean`, but after refactoring the `getOnlineForDepartment`, I'll use this method again.